_monitoring-plugins/alerting/api.md

```diff
@@ -508,17 +508,58 @@ POST _plugins/_alerting/monitors
   }
 }
 ```
-## Create a document-level monitor
+## Per document monitors
 Introduced 2.0
 {: .label .label-purple }
 
-You can create a document-level monitor programmatically that functions the same as a per document monitor in OpenSearch Dashboards. The document-level monitor runs a query that checks whether or not the results should trigger an alert notification based on individual documents within an index.
+Per document monitors check whether or not individual documents in an index match trigger conditions to generate an alert notification. When you run a query with a per document monitor, the results are returned for each document that matches the trigger condition. You can create trigger conditions based on query names, query IDs, or tags that combine multiple queries. 
+
+The following table shows the syntax to use for each trigger option:
+
+Trigger options | Definition | Syntax
+:--- | :--- : :---
+Tag | Creates alerts for documents that match a muliple query with this tag applied. If you group multiple queries by a single tag, then you can set it to trigger an alert if the results are returned by this tag name.| `query[tag=<tag-name>]`
+Query by name | Creates alerts for documents matched or returned by the named query.  | `query[name=<query-name>]`
+Query by ID | Creates alerts for documents that were returned by the identified query. | `query[id=<query-id>]`
+
+For more information about per document monitors, see [Monitor types]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/monitors/#monitor-types).
+
+### Search for monitor findings
+
+You can search the findings index `.opensearch-alerting-finding*` for available document findings with a GET request to the findings resource:
 
-You can combine multiple queries in a per document monitor by adding a tag to each query and then setting the trigger condition to that tag.
+```json
+GET /_plugins/_alerting/findings/_search?
+```
+The response returns all findings available.
```

Comment:
Can a finding be unavailable? Or does it just return every finding in that system index? If so, I would change this to, "By default, the request returns all findings."